urlshort/main/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	"go-exercises/urlshort"
+)
+
+func main() {
+	mux := defaultMux()
+
+	// Build the MapHandler using the mux as the fallback
+	pathsToUrls := map[string]string{
+		"/urlshort-godoc": "https://godoc.org/github.com/gophercises/urlshort",
+		"/yaml-godoc":     "https://godoc.org/gopkg.in/yaml.v2",
+	}
+	mapHandler := urlshort.MapHandler(pathsToUrls, mux)
+
+	//     // Build the YAMLHandler using the mapHandler as the
+	//     // fallback
+	yaml := `
+- path: /urlshort
+  url: https://github.com/gophercises/urlshort
+- path: /urlshort-final
+  url: https://github.com/gophercises/urlshort/tree/solution
+`
+	yamlHandler, err := urlshort.YAMLHandler([]byte(yaml), mapHandler)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Unless you are running test only kindly avoid `panic(err)` It returns too much information which can affect UX of your product.  Consider using 
```
if err != nil {
   fmt.Println(err)
   os.Exit(1)
}
``` 
exit with status 1 indicates an error occurred

Reply:
noted